tasks/rewrite-pug.js

```diff
@@ -0,0 +1,42 @@
+let glob = require(`glob`)
```

Comment:
Do we still need the converters for pug - stylus? 
I understood that the merge would remove this dependency, or it will take some more time?

Just can't wait to remove two dependencies from our package.json :)